src/main/java/com/iota/iri/service/dto/GetNeighborsResponse.java

```diff
@@ -44,6 +44,7 @@ public static Neighbor createFrom(com.iota.iri.network.Neighbor n) {
             Neighbor ne = new Neighbor();
             int port = n.getPort();
             ne.address = n.getAddress().getHostString() + ":" + port;
+            ne.ip = n.getAddress().getAddress().toString().split("/")[1];
```

Comment:
Please put `n.getAddress()` in a local variable, and use for the `ne.address`, `ne.ip` assignment.
This is because:
1. I don't like the concatenated double `getAddress()`.
2. In general, concatenated gets make it a bit harder to pinpoint null pointer exceptions.